extensions/2.0/Khronos/KHR_texture_video/README.md

```diff
@@ -0,0 +1,159 @@
+# KHR_texture_video
+
+## Contributors
+
+* Dwight Rodgers, Adobe
+* Gary Hsu, Microsoft
+* Emmett Lalish, Google
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension allows glTF models to use videos as textures. A client that does not implement this extension can ignore the provided video and continue to rely on the PNG and JPG textures available in the base specification. Defining a fallback texture is optional. 
+
+Video textures support use cases such as water or other textures that continuously repeat, coordination of video texture playback by animation (in conjunction with `KHR_animation_pointer`), or control of video texture playback by interactivity (in conjunction with `KHR_interactivity`).
+
+## glTF Schema Updates
+
+Videos, for use as textures, are defined in the glTF scene description file by adding an `extensions` property to the top level glTF 2.0 object and defining a `KHR_texture_video` property with a `videos` array inside it. Each object in the `videos` array provides the URI of a video texture.
+
+The `KHR_texture_video` extension is then added in an `extensions` property of each `textures` node that will use a video texture. The `source` property of the `KHR_texture_video` object points to the index of the video texture within the aforementioned `videos` object.
+
+The following glTF will load `texture.webm` in clients that support this extension, and otherwise fall back to `image.png`.
+
+```
+"textures": [
+    {
+        "source": 0,
+        "extensions": {
+            "KHR_texture_video": {
+                "source": 0,
+                "playhead": 0,
+                "autoplay": true,
+                "loop": true
+            }
+        }
+    }
+],
+"extensions": {
+    "KHR_texture_video": {
+        "videos": [
+            "uri": "texture.webm"
+        ]
```

Comment:
This is still not valid JSON.

Reply:
Indeed. Thanks, it believe it's fixed in the latest commit.